Support modern quickcheck (>0.8.0)

Modern quickcheck and quickcheck_macros do not require compiler plugin, but also have some small changes. So current implementation doesn't support it completely:
```
error[E0277]: the trait bound `petgraph::prelude::StableGraph<(), ()>: quickcheck::Arbitrary` is not satisfied
  --> radeco-lib/src/backend/ctrl_flow_struct/graph_utils/test.rs:12:1
   |
12 | #[quickcheck]
   | ^^^^^^^^^^^^^ the trait `quickcheck::Arbitrary` is not implemented for `petgraph::prelude::StableGraph<(), ()>`
   |
   = note: required because of the requirements on the impl of `quickcheck::Testable` for `fn(petgraph::prelude::StableGraph<(), ()>, usize, std::vec::Vec<usize>) -> quickcheck::TestResult`
   = note: required by `quickcheck::quickcheck`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,7 +27,7 @@ debug = true
 
 [dependencies]
 fixedbitset = { version = "0.1.4" }
-quickcheck = { optional = true, version = "0.7.1", default-features = false }
+quickcheck = { optional = true, version = "0.8", default-features = false }
 indexmap = { version = "1.0.2", optional = true }
 serde = { version = "1.0", optional = true }
 serde_derive = { version = "1.0", optional = true }

--- a/serialization-tests/Cargo.toml
+++ b/serialization-tests/Cargo.toml
@@ -19,6 +19,6 @@ itertools = { version = "0.6.2" }
 
 [dev-dependencies]
 serde_json = "1.0"
-quickcheck = { version = "0.7.1", default-features = false }
+quickcheck = { version = "0.8", default-features = false }
 bincode = "1.0.1"
 defmac = "0.1"

